_config.yml

```diff
@@ -0,0 +1,34 @@
+# Site settings
+title: "Open Science Paper"
+description: "A modern web framework for publishing true Open Science papers on GitHub."
+keywords: "HTML, CSS, JS"
+url: ""
+baseurl: "/test_gh-pages"
+
+
+# authors of the publication
+authors:
```

Comment:
Avoid putting your own email in repositories, robots might use it for spamming you afterwards.